package.json

```diff
@@ -157,10 +159,10 @@
     "request": "^2.81.0",
     "resin-cli-form": "^2.0.1",
     "resin-cli-visuals": "^1.4.0",
-    "resin-compose-parse": "^2.0.0",
+    "resin-compose-parse": "^2.0.3",
     "resin-doodles": "0.0.1",
     "resin-image-fs": "^5.0.2",
-    "resin-multibuild": "^2.1.2",
+    "resin-multibuild": "^2.1.4",
```

Comment:
The several version increases in `package.json` were simply matching the versions found in `resin-multibuild` for the respective packages. Apart from `resin-lint`, the rest probably does not make any difference.